pkg/controller/container-runtime-config/helpers.go

```diff
@@ -176,6 +188,10 @@ func getManagedKeyReg(pool *mcfgv1.MachineConfigPool) string {
 	return fmt.Sprintf("99-%s-%s-registries", pool.Name, pool.ObjectMeta.UID)
 }
 
+func getManagedKeyCaps(pool *mcfgv1.MachineConfigPool) string {
+	return fmt.Sprintf("99-%s-crio-capabilities", pool.Name)
```

Comment:
Minor: maybe we should a "-generated" or something like that as a suffix so users know that it was the controller who created it (since this won't have a controller version attached)

Reply:
yup, fixed!